opacus/tests/multigpu_gradcheck.py

```diff
@@ -68,36 +69,41 @@ def demo_basic(rank, weight, world_size, dp):
 
     # create model and move it to GPU with id rank
     model = ToyModel().to(rank)
+    optimizer = optim.SGD(model.parameters(), lr=1)
```

Comment:
Can't comment on unchanged lines!! Ugh! I hate github's restrictions.
nit: remove L38

Reply:
Also, the test won't run on win32 in its current form, right? due to L27?

Reply:
good catch: shall I just delete win32 part? I don't think we intend this as a public tutorial, as I presume our test env will always stay unix

Reply:
line of reasoning: you need manual intervention for running this on windows (provide a filename) -> it's not worth having as a test

Reply:
> you need manual intervention for running this on windows (provide a filename)

Wow! If it can't be automated and not a tutorial, I think we should remove it. If that portion of the code never runs, we will never know if it is doing its job correctly or not.

Curious though, is there no way to run automated tests on windows?

Reply:
No first hand experience, but my impression from reading the docs is that you need to provide a filename for syncronization, and windows don't have a conventional `/tmp` folder